### PR TITLE
The source code can be run by Rust Playground

### PR DESCRIPTION
**Pull request by java1cprog**
The source code can be run by Rust Playground without warnings

- [ ] I solemnly swear that this is all original content of which I am the original author
- [ ] Pull request title is prepended with `[language/lang-code]`
- [ ] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by divayprakash** [comment]
@java1cprog Thank you for the contribution and welcome to the community!

I understand your changes are intended to make the code directly runnable by the Rust Playground. However, the addition of the `break` statements in the infinite `while` and `loop` constructs rather defeats the purpose of the statements. Accordingly, could you please add comments clarifying the situation in the appropriate lines?

File: rust.html.markdown [hunk 1]
```diff
@@ -256,11 +260,13 @@ fn main() {
     // `while` loop
     while 1 == 1 {
         println!("The universe is operating normally.");
+        break;
     }
 
     // Infinite loop
     loop {
         println!("Hello!");
+        break;
```
Comment (divayprakash) [inline rust.html.markdown]: I understand your changes are intended to make the code directly runnable by the Rust Playground. However, the addition of the `break` statement here rather defeats the purpose of the statement. Accordingly, could you please add a comment clarifying the situation?

Comment (java1cprog) [reply rust.html.markdown]: Yes. I will add commentary about using break.

**Comment by divayprakash** [comment]
LGTM!

**Pull request merged**